### PR TITLE
update to latest wmcore

### PR DESCRIPTION
**Pull request by belforte**

### CODE REVIEW

File: crabserver.spec [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+### RPM cms crabserver 3.3.1809.rc3
 
 ## INITENV +PATH PATH %i/xbin
 ## INITENV +PATH PYTHONPATH %i/${PYTHON_LIB_SITE_PACKAGES}
 ## INITENV +PATH PYTHONPATH %i/x${PYTHON_LIB_SITE_PACKAGES}
 
 
 %define webdoc_files %{installroot}/%{pkgrel}/doc/
-%define wmcver 1.1.2.crab4
+%define wmcver 1.1.14.crab1
```
Comment (amaltaro) [inline crabserver.spec:9]: we discussed using the latest cmsweb production tag, so 1.1.14.pre4 :)

**Comment by belforte** [comment]
But 1.1.14.crab1 is what I tested in my VM and I do not want to start over again :-)

**Comment by h4d4** [comment]
-1 to 6f9861a4ba7b65d5fc58b73d6315c6dbdc0665ed on slc6_amd64_gcc493.<br>Build log can be found <a href="https://cern.ch/cms-dmwm-builds/4197/test_6f9861a-20180718-134508.log" target="_blank">here</a>.

**Comment by amaltaro** [comment]
That's fine, they should be very similar.
If you want to proceed as we discussed, that would be:
*) for crabserver: use the latest production WMCore release (the one in 
https://cmsweb.cern.ch/reqmgr2/data/info
*) for taskworker: keep using the latest tag available in crab branch (1.14.crab1 in this case)

**Comment by belforte** [comment]
Lina, I saw the message about the failure to build, but the link to logs is not working, points to an error page. I only changed version numbers, so have no idea what could be wrong.

**Comment by belforte** [comment]
Lina, I saw the message about the failure to build, but the link to logs is not working, points to an error page. I only changed version numbers, so have no idea what could be wrong.

**Comment by belforte** [comment]
@amaltaro  yes. Future will always be better. But this time I had started w/o talking with you first. sorry.

**Comment by h4d4** [comment]
@belforte 
Hi Stefano, last github output for this PR is that crabserver.spec is a conflicting file.
Please, try to access below:
https://github.com/cms-sw/cmsdist/pull/4197/conflicts

Best Regards, Lina.

**Comment by belforte** [comment]
no idea why the merge did not work. But should be fixed now.

**Comment by h4d4** [comment]
+1 to 7aa12573356a63266f30f2d50a6857fef1d52fbb on slc6_amd64_gcc493.<br>Build log can be found <a href="https://cern.ch/cms-dmwm-builds/4197/test_7aa1257-20180718-182507.log" target="_blank">here</a>.

**Pull request merged**